doc/source/examples/anchor_text_movie.ipynb

```diff
@@ -14,6 +14,20 @@
     "In this example, we will explain why a certain sentence is classified by a logistic regression as having negative or positive sentiment. The logistic regression is trained on negative and positive movie reviews."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "To enable support for the anchor text language models, you may need to run\n",
+    "    \n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?